Include/object.h

```diff
@@ -332,7 +332,8 @@ static inline Py_ssize_t Py_SIZE(PyObject *ob) {
 #  define Py_SIZE(ob) Py_SIZE(_PyObject_CAST(ob))
 #endif
 
-static inline Py_ALWAYS_INLINE int _Py_IsImmortal(PyObject *op)
+#if !defined(Py_LIMITED_API) || Py_LIMITED_API+0 < 0x030d0000
```

Comment:
Hmm, I'm not sure, but should it rather be:
```suggestion
#if defined(Py_LIMITED_API) && Py_LIMITED_API+0 >= 0x030d0000
PyAPI_FUNC(int) _Py_IsImmortal(PyObject *op);
#else
```

Reply:
There is no need to export this function in the stable ABI. Extract of the commit message:

> _Py_IsImmortal(), _Py_IsOwnedByCurrentThread() and _Py_ThreadId() have to be converted to an inline functions, since non-static Py_SET_REFCNT() function cannot call a static functions. _Py_IsImmortal(), _Py_IsOwnedByCurrentThread() and _Py_ThreadId() are **not part of the stable ABI**.

This function is an implementation detail. In the non-limited API, we have to implement expose it in the API (as a private function) and implement it as an inline function, since it's called by Py_SET_REFCNT() which is implemented as an inline function (in the non-limited API).

In limited C API version 3.13, Py_SET_REFCNT() is called as an opaque function call which calls internally _Py_IsImmortal() (and _Py_IsOwnedByCurrentThread() and _Py_ThreadId() in free threading build).